web/modules/custom/usagov_login/src/Form/LoginSettingsForm.php

```diff
@@ -47,22 +46,32 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
-  public function getFormId() {
+  public function getFormId(): string {
     return 'usagov_login_login_settings';
   }
 
   /**
    * {@inheritdoc}
+   *
+   * @return string[] Array of editable configuration names.
    */
-  protected function getEditableConfigNames() {
+  protected function getEditableConfigNames(): array {
     return ['usagov_login.settings'];
   }
 
   /**
-   * {@inheritdoc}
+   *  {@inheritdoc}
+   *
+   * @param array<string, mixed> $form
+   *    The form array to be built.
+   * @param FormStateInterface $form_state
+   *    The current state of the form.
```

Comment:
Let's not add docblock type hints that are the same as the native typehints. If the PHP type hint changes, this one has to change to keep up.